data/json/items/ammo.json

```diff
@@ -946,6 +946,7 @@
         "category" : "chems",
         "price" : 3500,
         "name" : "sprayable fungicide",
+        "name_plural" : "sprayable fungicide",
```

Comment:
Then we need to revise the string extraction script to let it know which items are stored by charges while which are stored by entries. This is a little bit difficult for me that I don't know much about Python. This PR is a quick fix to eliminate these two incorrect strings from translation templates. The exhaustive fix will be implemented later.